docs/sql-ref-syntax-aux-analyze-table.md

```diff
@@ -19,4 +19,45 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+ANALYZE TABLE statement can be used to collect statistics about the table. The information can be used by the query optimizer to find a better plan.
```

Comment:
Back-tick the reserved words for clarity.
plan -> query plan? maybe query execution plan?